### PR TITLE
Fix payment flow

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: server/api/payment.js [hunk 1]
```diff
@@ -133,6 +133,21 @@ router.post('/payment', async (req, res, next) => {
       LIKECOIN.LIKE_COIN_ADDRESS,
       txData,
     );
+    if (!toSubscriptionURL) {
+      res.json({ txHash });
+    }
```
Comment (williamchong007) [inline server/api/payment.js]: sending both `res.json({ txHash });` here and below in 168 will cause error, on the other hand, why need to send early？

Comment (nwingt) [reply server/api/payment.js]: Updated. This make sure sending res after `logTransferDelegatedTx` if user has `toSubscriptionURL`. If send earlier than that, oice cannot query the tx through tx API

Comment (williamchong007) [reply server/api/payment.js]: just defer the `res.json({ txHash })` after `logTransferDelegatedTx` i guess

Comment (nwingt) [reply server/api/payment.js]: Updated

**Pull request merged**